resolwe/flow/utils/__init__.py

```diff
@@ -24,7 +25,7 @@
 def get_data_checksum(proc_input, proc_slug, proc_version):
     """Compute checksum of processor inputs, name and version."""
     checksum = hashlib.sha1()
```

Comment:
Probably not, I will change it to SHA256 as we already have to update all checksums